app/src/test/java/com/codesoom/assignment/controllers/TaskControllerTest.java

```diff
@@ -0,0 +1,212 @@
+package com.codesoom.assignment.controllers;
+
+import com.codesoom.assignment.TaskNotFoundException;
+import com.codesoom.assignment.application.TaskService;
+import com.codesoom.assignment.models.Task;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.test.web.servlet.MockMvc;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.BDDMockito.given;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
+
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.post;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.content;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+@SpringBootTest
+@AutoConfigureMockMvc
+@DisplayName("TaskController 클래스")
+class TaskControllerTest {
+    @Autowired
+    private MockMvc mockMvc;
+
+    @MockBean
+    private TaskService taskService;
+
+    private final long TASK_ID = 0;
+    private final String TASK_TITLE = "sample";
+
+    @Nested
+    @DisplayName("list 메소드는")
+    class Describe_list {
+
+        @Nested
+        @DisplayName("Task 가 하나도 없을 때")
```

Comment:
`@BeforeEach`를 읽어보면 다음과 같이 메시지를 수정하는 것도 생각해볼 수 있을 것 같아요.

```suggestion
        @DisplayName("서비스에 등록된 Task 가 하나도 없을 때")
```

Reply:
아, 그러고보니 서비스에서 불러오는 구조이니 그게 맞겠군요!

Reply:
`등록된`도 좋을 것 같아요. `서비스에 등록된`은 `@Service 클래스에 등록된` 으로 착각될 가능성이 있을 것 같습니다.